lib/shoulda/matchers/active_model.rb

```diff
@@ -1,5 +1,6 @@
 require 'shoulda/matchers/active_model/helpers'
 require 'shoulda/matchers/active_model/validation_matcher'
+require 'shoulda/matchers/active_model/validator'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.